app/views/proposals/index.html.haml

```diff
@@ -116,4 +116,5 @@
   .row
     .col-md-12
       - if can? :create, @event
-        = link_to "New Proposal", new_conference_program_proposal_path(@conference.short_title), class: 'btn btn-success pull-right'
+        - if @program.cfp_open?
```

Comment:
How can this be if the policy is already checking this?

```
can [:new, :create], Event do |event|
    event.program.cfp_open? && event.new_record?
end
```

Reply:
@hennevogel  Yes, you are right. I read about cancan after your comment, we are already checking this :see_no_evil: ,  only admins will face this no method error problem on "submit proposal". I will remove the last commit. 
Thanks

Reply:
but the same is true for the first commit right?

Reply:
@hennevogel Yes, I know :smile: . I am working on the fix for the bug and will modify this commit.